pro/src/ui-kit/Table/Table.tsx

```diff
@@ -0,0 +1,339 @@
+        e.preventDefault()
+        handleRowNavigation(row)
+      }
+    }
+
+  return (
+    <div className={classNames(styles.wrapper, className)} tabIndex={0}>
+      {selectable && (
+        <div className={styles['table-select-all']}>
+          <Checkbox
+            label={
+              selectedIds.size < selectableRows.length
+                ? 'Tout sélectionner'
+                : 'Tout désélectionner'
+            }
+            checked={
+              selectedIds.size === selectableRows.length &&
+              selectableRows.length > 0
+            }
+            onChange={toggleSelectAll}
+            indeterminate={
+              selectedIds.size > 0 && selectedIds.size < selectableRows.length
+            }
+          />
+          <span className={styles['visually-hidden']}>
+            Sélectionner toutes les offres
+          </span>
+        </div>
+      )}
+
+      <table role="table" className={styles.table}>
+        <caption className={styles['visually-hidden']}>{title}</caption>
+        <thead>
+          <tr
+            className={classNames(styles['table-header'], {
+              [styles['table-header']]: isSticky,
+            })}
+          >
+            {selectable && (
+              <th className={styles['table-header-th']}>
+                <span className={styles['visually-hidden']}>
+                  Sélectionner les lignes
+                </span>
+              </th>
+            )}
+            {columns.map((col) => {
+              if (col.headerHidden) {
+                return null
+              }
+              return (
+                <th
+                  id={col.id}
+                  scope="col"
+                  colSpan={col.headerColSpan || 1}
+                  key={col.id}
+                  className={classNames(
+                    styles.columnWidth,
+                    styles['table-header-th'],
+                    {
+                      [styles.sortable]: col.sortable,
+                    }
+                  )}
+                >
+                  {col.label}
+                  {col.sortable && (
+                    <SortArrow
+                      onClick={() => toggleSort(col.id, col.sortable)}
+                      sortingMode={sortDir}
+                    />
+                  )}
+                </th>
+              )
+            })}
+          </tr>
+        </thead>
+
+        <tbody role="rowgroup">
+          {isLoading &&
+            Array.from({ length: 8 }).map((_, index) => (
+              <tr key={`loading-row-${index}`}>
+                <td colSpan={columns.length + 1}>
+                  <Skeleton height="7rem" width="100%" margin="0" />
+                </td>
+              </tr>
+            ))}
+
+          {sortedData.map((row) => {
+            const isSelected = selectedIds.has(row.id)
+            const expandedContent = getExpandedContent?.(row)
+            const hasNavigation = Boolean(getRowLink?.(row))
+
```

Comment:
Pas possible de faire ça ?
```suggestion
                <tr  key={row.id}
```